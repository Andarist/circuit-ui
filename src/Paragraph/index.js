import React from 'react';
import PropTypes from 'prop-types';
import withStyles from '../../util/withStyles';
import styles from './index.scss';

const Paragraph = ({ children, ...props }) => {
  const classes = 'paragraph';
  return (
    <p className={classes} {...props}>
      {children}
    </p>
  );
};

Text.propTypes = {
  children: PropTypes.node
};

export default withStyles(styles)(Paragraph);
