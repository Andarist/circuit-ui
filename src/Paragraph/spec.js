import React from 'react';
import renderer from 'react-test-renderer';
import Paragraph from '.';

describe('Paragraph', () => {
  it('should render a paragraph', () => {
    const actual = renderer.create(<Paragraph>Hello</Paragraph>);
    expect(actual).toMatchSnapshot();
  });
});
