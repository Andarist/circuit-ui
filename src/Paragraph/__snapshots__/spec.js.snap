// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph should render a paragraph 1`] = `
<p
  className="paragraph"
>
  Hello
</p>
`;
